components/partials/fee-discounts/fees/index.vue

```diff
@@ -101,8 +101,7 @@ export default Vue.extend({
           makerDiscountRate: '0',
           takerDiscountRate: '0',
           stakedAmount: '0',
-          feePaidAmount: '0',
-          volume: '0'
+          feePaidAmount: '0'
```

Comment:
Why remove `volume`?

Reply:
the build was complaining, just added it back after updating the `sdk-ui` and `sdk-ui-ts` package 👍